src/searching.md

```diff
@@ -35,24 +34,24 @@ finds notes with neither "cat" nor "mouse".
 same as the above.
 
 `"a dog"`\
-finds notes with the exact sequence of characters "a dog" on them, such
+finds notes with the exact sequence of characters "a dog" in them, such
 as "atta dog", but not "dog a" or "adog".
 
 `-"a dog"`\
-finds notes without the exact phrase "a dog"
+finds notes without the following sequence of characters: "a dog".
```

Comment:
Since this is offered in contrast to the example above, better to mirror that line: 
`finds notes without the exact sequence of characters "a dog" in them.`

Reply:
I have mentioned below that some of this is intentional. This section/page of the manual has a lot of similar sounding sentences and it can quickly get boring to read through this. A little bit of variability, that catches the reader off guard, can draw their attention to what is written in the doc. I do value being consistent in formatting and else, but some amount of entropy is nice to have.

Reply:
> some amount of entropy is nice to have.

In the manual, or in any technical writing? -- I strongly disagree. This isn't prose. Parallel phrasing is how you quickly and efficiently communicate similarity and subtle differences between things. 

Adding variability makes things harder for a reader to understand at-a-glance, and ruins the effect of contrasting the examples in the first place. Different wording/terminology should be used to signal for a reader that the manual is talking about about something different -- because that's the impression it gives.

Reply:
```suggestion
finds notes without the exact sequence of characters "a dog" in them.
```